### DISCUSSION TITLE
Wagmi/core does work with TypeScript strict mode

### DISCUSSION THREAD

**Discussion by HufsaEu** [wevm/wagmi#2605] [Q&A]
Hello, 
I have a problem using @wagmi/core. After adding it to my Angular 16 application it starts throwing a lot of errors in `node_modules/@wagmi/connectors/dist/injected.d.ts` file.

<img width="920" alt="Screenshot 2023-06-23 at 21 01 08" src="https://github.com/wagmi-dev/wagmi/assets/42934071/68bbb40f-0208-433a-9f9c-0d4db9aa60b2">

After investigating a bit it turns out disabling 'strict' mode in `tsconfig.json` does the trick and it builds without issues.

I think my tsconfig.json is a standard file for Angular app as I don't remember changing anything there:
```
/* To learn more about this file see: https://angular.io/config/tsconfig. */
{
  "compileOnSave": false,
  "compilerOptions": {
    "baseUrl": "./",
    "outDir": "./dist/out-tsc",
    "forceConsistentCasingInFileNames": true,
    "strict": true,
    "noImplicitOverride": true,
    "noPropertyAccessFromIndexSignature": true,
    "noImplicitReturns": false,
    "strictPropertyInitialization": false,
    "noFallthroughCasesInSwitch": true,
    "sourceMap": true,
    "declaration": false,
    "downlevelIteration": true,
    "experimentalDecorators": true,
    "moduleResolution": "node",
    "importHelpers": true,
    "resolveJsonModule": true,
    "esModuleInterop": true,
    "target": "ES2022",
    "module": "ES2022",
    "lib": [
      "ES2022",
      "dom"
    ],
    "useDefineForClassFields": false
  },
  "angularCompilerOptions": {
    "enableI18nLegacyMessageIdFormat": false,
    "strictInjectionParameters": true,
    "strictInputAccessModifiers": true,
    "strictTemplates": true
  }
}

```

Does anyone have any idea how to resolve this issue? TBH I don't really want to disable 'strict' mode so this is not an option.

**Comment by glitch-txs** [comment]
Have you tried adding "skipLibCheck" as true?

**Reply by HufsaEu** [reply]
Yeah it does work indeed. Definitely a better solution than disabling strict mode, and at least can work with it for now. Thanks 👍

**Reply by juventusjude** [reply]
SkipLibCheck helps but more problems I can't figure . I guess the team need to test on angular. 
Gonna send snippets of issues soon.

**Reply by juventusjude** [reply]
After setting SkipLibCheck true, I was able to solve the above issue but stuck on another error as shown in the picture below. I am using angular.
![error with wagmi](https://github.com/wagmi-dev/wagmi/assets/22204290/a504c053-57a0-4eef-aafa-731d9e1f44f1)

**Comment by llllvvuu** [comment]
Hi, can you see upgrading to wagmi 1.3.3 fixes the issue? https://github.com/wagmi-dev/references/pull/381
